addons/calendar/models/discuss_channel_member.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+class DiscussChannelMember(models.Model):
+    _inherit = 'discuss.channel.member'
+
+    def _should_invite_members(self):
+        if self.channel_id.calendar_event_id and fields.Datetime.now() < self.channel_id.calendar_event_id.start:
+            return False
```

Comment:
(all files) not sure about guideline in this module but in our files we would apply formatter (double quotes, not too long lines, etc)